src/components/test/Feed.test.tsx

```diff
@@ -139,4 +139,15 @@ describe("Feed", () => {
       });
     });
   });
+
+  it("displays link correctly in post", () => {
+    const component = mount(componentWithStore(Feed, store));
+    expect(
+      component.find(".Post__caption").first().prop("dangerouslySetInnerHTML")
+        ?.__html
+    ).toEqual(
+      'test post <a class="messageLink" href=https://www.unfinishedlabs.io/ target="_blank">https://www.unfinishedlabs.io/</a>'
```

Comment:
linter does not like this.  tried `` as well but prettier changed them to ''. any suggestions?

Reply:
@claireolmstead Let's update the .eslintrc.json file rules to this (which will fix it and allow `'I "like" this rule better'`):

OLD: `"quotes": ["error", "double"],`
NEW: `"quotes": ["error", "double", { "avoidEscape": true, "allowTemplateLiterals": true }],`